buffer/buffer.go

```diff
@@ -74,6 +74,12 @@ func (b *Buffer) AppendFloat(f float64, bitSize int) {
 	b.bs = strconv.AppendFloat(b.bs, f, 'f', -1, bitSize)
 }
 
+// AppendFloatWithPrecision appends a float with precision to the underlying buffer. It doesn't quote NaN
+// or +/- Inf.
+func (b *Buffer) AppendFloatWithPrecision(f float64, bitSize int, prec int) {
```

Comment:
`AppendFloat` takes `prec, bitSize int` as the last 2 arguments, let's do the same here for consistency